src/unnamed_rcv_thing/cleaning.py

```diff
@@ -0,0 +1,19 @@
+from unnamed_rcv_thing.profile import PreferenceProfile
```

Comment:
within the package, can omit package name (from .profile import PreferenceProfile) 

mentioned b/c it will help with the refactoring